src/firebase/config.js

```diff
@@ -20,8 +21,8 @@ const firebaseConfig = {
 	projectId: PROJECT_ID,
 	storageBucket: STORAGE_BUCKET,
 	messagingSenderId: MESSAGING_SENDER_ID,
-	appId: APP_ID,
-	measurementId: MEASUREMENT_ID
+	appId: APP_ID
+	// measurementId: MEASUREMENT_ID
```

Comment:
Why you choose to comment on this instead removing it?

Reply:
Ngl, I was cleaning/testing different config values and forget to delete `measurementId: MEASUREMENT_ID` before pushing the code